goosebit/updater/routes.py

```diff
@@ -12,14 +12,14 @@
 
 async def log_last_connection(request: Request, dev_id: str):
     device = await get_device(dev_id)
-    if config.track_device_ip:
+    if request.scope["config"].track_device_ip:
```

Comment:
Why do we sometimes import the config to use it and sometime we use the middleware to access it?

Reply:
Middleware is slightly more dynamic, and should be used in the routes where the `Request` can be accessed.  Everywhere else can use config because it doesn't need to be dynamic or it doesn't have access to request.